pkg/server/backgroundsvcs/background_services.go

```diff
@@ -46,7 +47,7 @@ func ProvideBackgroundServiceRegistry(
 	_ *azuremonitor.Service, _ *cloudwatch.CloudWatchService, _ *elasticsearch.Service, _ *graphite.Service,
 	_ *influxdb.Service, _ *loki.Service, _ *opentsdb.Service, _ *prometheus.Service, _ *tempo.Service,
 	_ *testdatasource.TestDataPlugin, _ *plugindashboards.Service, _ *dashboardsnapshots.Service,
-	_ *postgres.Service, _ *mysql.Service, _ *mssql.Service,
+	_ *postgres.Service, _ *mysql.Service, _ *mssql.Service, _ *grafana.Service,
```

Comment:
are we at all worried about the name "grafana.Service" as the builtin plugin name?  seems like the scope could be too large.  maybe `grafanads.Service`?

Reply:
I am renaming to `grafana_ds.Service` -- I don't see any other `_` but `grafanads` reads to me like "grafa-nads" :grimacing: